docs/config.md

```diff
@@ -0,0 +1,19 @@
+# OSM ConfigMap
+
+OSM deploys a configMap `osm-config` as a part of its control plane (in the same namespace as that of the osm-controller pod) which can be updated by the mesh owner/operator at any time. The purpose of this configMap is to provide the mesh owner/operator the ability to reconfigure/update some of the mesh configurations based on their needs. 
+
+## ConfigMap Values
+
+| Key | Type | Possible Values | Default Value | Function |
+|-----|------|-----------------|---------------|----------|
+| permissive_traffic_policy_mode | bool | true, false | `"false"` | Setting to `true`, enables allow-all mode on the mesh i.e. all services in the mesh will be able to talk to one and other. If set to `false`, enables deny-all on mesh i.e. an `SMI Traffic Target` is necessary for a service in the mesh to talk to another service in the mesh. |
+| egress | bool | true, false| `"false"` | Enables egress on the mesh. |
+| enable_debug_server | bool | true, false| `"true"` | Enables the debug endpoint on the osm-controller pod to inspect and list most of the common structures used by the control plane at runtime. |
```

Comment:
What does `list most of the common structures` mean here?

Reply:
it refers to the structs/objects that OSM creates

Reply:
I would change this to something users can comprehend: `Enables a debug endpoint on the osm-controller pod to list information regarding the mesh such as proxy connections, certificates, and SMI policies, .